cli/src/code_coverage/prvsnr-coverage.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+import argparse
+import subprocess
+import sys
+import os.path
+import shlex
+
+def _run_command(cmd):
+    try:
+        res = subprocess.run(shlex.split(cmd), shell=False)
+        if res.returncode:
+            print(f"failed to run command {cmd} ")
+            sys.exit(1)
+    except Exception as e:
+        print(e)
+        sys.exit(1)
+
+def install_coverage():
+    print("Installing prerequisites")
+    _run_command(
+        f"yum install -y gcc cpp python3-devel"
+    )
+    _run_command(
+        f"pip3 install -U api/python"
+    )
+    _run_command(
+        f"pip3 install -r test-requirements.txt"
+    )
+
+def prvsnr_coverage():
+    print("Executing provisioner test cases and generating coverage report")
+    _run_command(
+        f"coverage run -m pytest \
+        --cov test/ --cov-report=xml"
+    )
+    if os.path.isfile(f'coverage.xml'):
+        print("Coverage report generated successfully!!")
+    else:
+        print("Coverage report generation failed!!")
+        sys.exit(1)
+    print("Done")
+
+def parse_args():
```

Comment:
Is this a private method?
It would be better to have it as:
`_parse_args()`

Reply:
Addressed